src/components/Nav.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
+import './Nav.css';
```

Comment:
- [ ] It'll be good not to have commented-on code in your codebase as it makes your code unprofessional. It's a good convention to have our codebase clean and professional. Kindly remove all commented unused codes 👍🏽